Developers/Executing_Tests.rst

```diff
@@ -28,7 +28,7 @@ coala project directory.
 
 ::
```

Comment:
hey. first of all, your commit shortlog is huge: maximum is 50 chars we allow, please lower it :D

2ndly, in your body you shouldnt use past tense, use "Removes"

also, place a newline between body and the "Fixes" thingie

lastly, it is not a bug, so please use Closes url_link, instead of Fixes, and do not place a colon there :D thanks